Client/Application/AppDelegate.swift

```diff
@@ -78,6 +83,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         ReaderModeHandlers.register(server)
         server.start()
     }
+
+    private func setUserAgent() {
+        let webView = UIWebView()
+        let userAgent = webView.stringByEvaluatingJavaScriptFromString("navigator.userAgent")!
```

Comment:
I'll add a simple test to make sure the app's UA matches the pattern we expect from [bug 1147658](https://bugzilla.mozilla.org/show_bug.cgi?id=1147658).